pages/about.tsx

```diff
@@ -3,7 +3,7 @@ import ModelInfo from '../components/ModelInfo'
 import models, {ModelSpec} from '../lib/models'
 import styles from './about.module.css'
 
-export default function IndexPage() {
+export default function AboutPage() {
```

Comment:
**Question:** This seems unrelated to the task at hand, but useful?

Reply:
Unrelated. Just a change that should have happened long ago. I'll extract to a different commit.